tests/test_cli.py

```diff
@@ -161,30 +160,6 @@ def test_cli_help_option():
     ) in result.output
 
 
-def test_cli_auth_command_usage():
```

Comment:
Linting complained that this file was too long after my additions, so I broke out all of the usage tests into a separate file.

Reply:
I can only assume the test file will continue to grow and the lint was configured to keep files readable, but I'm happy to go whichever way you would like.

Reply:
You assumption sounds right to me. Thanks for this initiative.